src/eureka-client.js

```diff
@@ -204,18 +205,16 @@ export class Eureka {
     if (!registry) {
       throw new Error('Unable to transform empty registry');
     }
-    if (registry.length) {
```

Comment:
Well, now that I've taken another look, it's silently broken anyway. When there's no instances `applications.application` is undefined, when theres 1 application, `applications.application` is an object, AND when there's more than 1 application, `applications.application` is an array. Can you change the `if` statement to `registry.applications.application` and then add another check for `registry.applications.application.length` and process a single application if that's undefined?